packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue

```diff
@@ -68,15 +68,21 @@
           >
             <div class="sf-product-card__add-button--icons">
               <transition name="sf-pulse" mode="out-in">
-                <slot v-if="!isAddingToCart" name="add-to-cart-icon">
+                <slot
+                  v-if="!isAddingToCart && !isAddedToCart"
+                  name="add-to-cart-icon"
+                >
                   <SfIcon
                     key="add_to_cart"
                     icon="add_to_cart"
                     size="20px"
                     color="white"
                   />
                 </slot>
-                <slot v-else name="adding-to-cart-icon">
+                <slot
+                  v-if="isAddingToCart || isAddedToCart"
```

Comment:
put this one before the other `slot` and set the other slot condition to `v-else` instead of repeating the condition just in the opposite condition

Reply:
Something similar to 
```
<slot
    v-if="isAddingToCart || isAddedToCart"
    name="adding-to-cart-icon"
>
</slot>
<slot v-else
 name="add-to-cart-icon
>
</slot>
```
You don't need two `v-if`